kitchen.vagrant.yml

```diff
@@ -0,0 +1,15 @@
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  deprecations_as_errors: true
+  chef_license: accept-no-persist
+  chef_client_path: /usr/bin/chef-client
+
+verifier:
+  name: inspec
+
+platforms:
+  - name: freebsd-11
+  - name: freebsd-12
```

Comment:
This shouldn't be needed. This should just work in `kitchen.yml` instead. Can you please make that change?